### PR TITLE
Consistent Community 404 Illustration

### PR DESCRIPTION
**Pull request by ttseng**
## Links
* Remix link: https://community-404.glitch.me
* Issue-tracker link: https://github.com/FogCreek/Glitch-Community-Work/issues/249

## GIF/Screenshots:
Replaced compass illustrations + telescope with empty aquarium animation for more consistent 'not found' UX

aquarium

## Changes:
* Page Not Found page (https://community-404.glitch.me/awegoik)
image

* Filtering on user profiles:
image

* No search results:
image

## How To Test:
* Check the 404 page (https://community-404.glitch.me/awegoik)
* Go to a user profile page (https://community-404.glitch.me/@scientiffic) and try filtering collections or projects by name
* Search for something that doesn't have search results (https://community-404.glitch.me/@scientiffic)

### CODE REVIEW

File: src/components/errors/not-found.js [hunk 1]
```diff
@@ -1,18 +1,82 @@
...
 import PropTypes from 'prop-types';
+import classnames from 'classnames';
 
 import Text from 'Components/text/text';
 import styles from './not-found.styl';
 
-const compass = 'https://cdn.glitch.com/bc50f686-4c0a-4e20-852f-3999b29e8092%2Fcompass.svg?1545073264846';
-const needle = 'https://cdn.glitch.com/bc50f686-4c0a-4e20-852f-3999b29e8092%2Fneedle.svg?1545073265096';
+export const Aquarium = ({ className }) => (
+  <svg className={classnames(styles.aquarium, className)} viewBox="0 0 88 86" xmlns="http://www.w3.org/2000/svg">
```
Comment (sarahzinger) [inline src/components/errors/not-found.js]: I think we need either an aria-labelled by or a <title> element here? but not 100% on that

Comment (clottman) [reply src/components/errors/not-found.js]: Yep! `aria-label=''` would be ideal so it indicates to screenreader users that this is just decorative. 

`<title>` works too but some browsers show the title as a tooltip, which is probably not what we want.

File: src/presenters/pages/error.js [hunk 2]
```diff
@@ -30,14 +28,13 @@ export const NotFoundPage = () => {
     <Layout>
       <Helmet title="👻 Page not found" />
       <main className={styles.container}>
```
Comment (clottman) [inline src/presenters/pages/error.js]: I missed this page when adding skip links - would you mind adding `id='#main'` to the `<main>` element?

File: src/components/errors/not-found.js (hunk 1 above)
Comment (ttseng) [reply src/components/errors/not-found.js]: got it!  added that in

File: src/presenters/pages/error.js (hunk 2 above)
Comment (ttseng) [reply src/presenters/pages/error.js]: sure - added to all the error pages

**Pull request merged**